backend/app-orchestrator-service/api/src/main/kotlin/api/Compute.kt

```diff
@@ -89,6 +89,23 @@ data class ComputeOpenInteractiveSessionRequestItem(
 )
 data class ComputeOpenInteractiveSessionResponse(val sessions: List<OpenSession>)
 
+typealias UtilizationRequest = Unit
+data class UtilizationResponse(
+    val allocatable: CpuAndMemory,
+    val used: CpuAndMemory,
+    val jobs: JobUtilization
```

Comment:
I think it would make more sense if this was renamed to:

```suggestion
    val capacity: CpuAndMemory,
    val usedCapacity: CpuAndMemory,
    val queueStatus: QueueStatus
```